src/Symfony/Component/HttpFoundation/File/UploadedFile.php

```diff
@@ -214,7 +214,8 @@ public function move($directory, $name = null)
      */
     public static function getMaxFilesize()
     {
-        $iniMax = strtolower(ini_get('upload_max_filesize'));
+        $iniMax = min([ini_get('upload_max_filesize'), ini_get('post_max_size')]);
```

Comment:
min() cannot work here, we should parse post_max_size before calling it, as it can be a string with units (e.g. 8M)
please also add test cases

Reply:
Your're right. Would it be okay to add an `parseFilesize` to the class? Also since there seems to be no test for `getMaxFilesize` till now I'm not entirely sure how to create one since it depends on the ini settings.

Reply:
> Would it be okay to add an parseFilesize to the class

a private one yes,

> not entirely sure how to create one since it depends on the ini settings

dunno if it's possible to use ini_set here... if not, I don't know either :)

Reply:
Well, I've implemented the proper conversion and check on the two values but regarding the tests it seems we're out of luck. We can't set e.g. `upload_max_filesize` using `ini_set()` because `upload_max_filesize` is a `PHP_INI_PERDIR` type which means it's only changeable via php.ini, .htaccess or httpd.conf ( see [php.net/manual/en/configuration.changes.modes.php](https://php.net/manual/en/configuration.changes.modes.php))

That also explains why there was no test on this method to begin with.